iac-extensions/kubernetes/src/main/java/org/sonar/iac/kubernetes/visitors/KubernetesChecksVisitor.java

```diff
@@ -109,6 +109,17 @@ protected void reportIssue(@Nullable TextRange textRange, String message, List<S
       }
     }
 
+    private HelmInputFileContext computeHelmInputFileContextForSecondaryLocation(SecondaryLocation secondaryLocation, HelmInputFileContext defaultHelmContext) {
```

Comment:
Maybe just `computeInputFileContext()`? The line 101 is very long because of this name.